e2e/generateDeployments.js

```diff
@@ -75,6 +75,12 @@ async function run() {
   const system = deployments.state['provision.system'].artifacts.imports.system;
   const contracts = {};
 
+  contracts.LegacyMarketProxy =
+    deployments.state['provision.legacyMarket'].artifacts.imports.legacyMarket.contracts.Proxy;
```

Comment:
This does not exist on any other deployments. It needs to be conditioned. Also why is it added first and not last?